src/main/kotlin/no/nav/familie/tilbake/api/dto/BehandlingsstegDto.kt

```diff
@@ -56,3 +61,41 @@ data class ForeldelsesperiodeDto(val periode: PeriodeDto,
                                  val foreldelsesfrist: LocalDate? = null,
                                  val oppdagelsesdato: LocalDate? = null)
 
+
+@JsonTypeName(BehandlingsstegVilkårsvurderingDto.STEG_NAVN)
+data class BehandlingsstegVilkårsvurderingDto(val vilkårsvurderingsperioder: List<VilkårsvurderingsperiodeDto>)
+    : BehandlingsstegDto() {
+
+    override fun getSteg(): String {
+        return STEG_NAVN
+    }
+
+    companion object {
+
+        const val STEG_NAVN = "VILKÅRSVURDERING"
+    }
+}
+
+data class VilkårsvurderingsperiodeDto(
+        val periode: PeriodeDto,
+        val vilkårsvurderingsresultat: Vilkårsvurderingsresultat,
+        val begrunnelse: String,
+        val godTroDto: GodTroDto? = null,
+        val aktsomhetDto: AktsomhetDto? = null)
+
+data class GodTroDto(val beløpErIBehold: Boolean,
+                     val beløpTilbakekreves: BigDecimal? = null,
+                     val begrunnelse: String)
+
+data class AktsomhetDto(val aktsomhet: Aktsomhet,
+                        val ileggRenter: Boolean? = null,
+                        val andelTilbakekreves: BigDecimal? = null,
+                        val manueltSattBeløp: BigDecimal? = null,
+                        val begrunnelse: String,
+                        val særligGrunner: List<SærligGrunnDto>? = null,
+                        val særligeGrunnerTilReduksjon: Boolean = false,
+                        val tilbakekrevSmåbeløp: Boolean = true,
+                        val særligeGrunnerBegrunnelse: String? = null)
+
+data class SærligGrunnDto(val særligGrunn: SærligGrunn,
+                          val begrunnelse: String? = null)
```

Comment:
denne er berre aktuell for særlige grunner === ANNET. kunne den ikkje vært på AktsomhetDto som annettBegrunnelse?

Reply:
Og då trenger me ikkje ein SærligGrunnDto?

Reply:
diskuterte på slack. Det trenger vi